### PR TITLE
doc(release): CHANGELOG for v0.3.0

### PR DESCRIPTION
**Pull request by sh0rez**
Prepares release v0.3.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,12 +1,35 @@
...
+- **[FEATURE] Absolute imports (#63)**: Introduces a new style for importing the
+  packages installed by `jb`. The `<name>/<file>` style used before caused
+  issues, as it was neither unique nor clearly defined what to import.  
+  To address this, `jb` will now create a directory structure that allows to use
+  import paths similar to Go: `host.tld/user/repo/subdir/file.libsonnet`.  
+  The old stlye is still supported, this change is backwards compatible.  
+  `jb rewrite` can be used to automatically convert your imports.
+- **[FEATURE] Generic `git` `https://`**: Previously the `host.tld/user/repo` slug
+  style was only supported for GitHub. All hosts work now.
+- **[BUGFIX]** `--jsonnetpkg-home` not working
```
Comment (brancz) [inline CHANGELOG.md]: can you add the PR that this was fixed in?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,12 +1,35 @@
...
+#### Changes:
+
+- **[FEATURE] Absolute imports (#63)**: Introduces a new style for importing the
+  packages installed by `jb`. The `<name>/<file>` style used before caused
+  issues, as it was neither unique nor clearly defined what to import.  
+  To address this, `jb` will now create a directory structure that allows to use
+  import paths similar to Go: `host.tld/user/repo/subdir/file.libsonnet`.  
+  The old stlye is still supported, this change is backwards compatible.  
+  `jb rewrite` can be used to automatically convert your imports.
+- **[FEATURE] Generic `git` `https://`**: Previously the `host.tld/user/repo` slug
```
Comment (brancz) [inline CHANGELOG.md]: can you add the PR that this was added in?

**Comment by sh0rez** [comment]
@brancz PTAL

**Comment by metalmatze** [comment]
I'm wondering if we should add the `version: 3` to the jsonnetfile.json and jsonnetfile.lock.json before cutting a release. We should then also handle the case where `version != 3` and prompt users to update their jsonent-bundler, rather than panicking.

**Comment by metalmatze** [comment]
I've started an implementation here:
https://github.com/jsonnet-bundler/jsonnet-bundler/pull/85

**Comment by hangxie** [comment]
Bump ... any update on 0.3.0 release?

**Comment by metalmatze** [comment]
LGTM
Ready when you are!

**Pull request merged**